README.md

```diff
@@ -1,9 +1,22 @@
-# matter.js
+# JavaScript/TypeScript based Matter Implementation
 
 ![experimental](https://img.shields.io/badge/status-Experimental-red) [![license](https://img.shields.io/badge/license-Apache2-green.svg)](https://raw.githubusercontent.com/project-chip/matter.js/master/LICENSE)
 
 Implementation of Matter protocol in typescript with no native dependencies (and very limited dependencies).
 
+## Monorepo Overview
+
+This repository contains multiple packages (and because of this it is a monorepo). The packages are contained in the `packages` directory and arer all published separately to NPM.
```

Comment:
```suggestion
This repository contains multiple packages (and because of this it is a monorepo). The packages are contained in the `packages` directory and are all published separately to NPM.
```